decentraland/components/presentational/__snapshots__/AccountCreationProgress.test.js.snap

```diff
@@ -1,24 +1,106 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`AccountCreationProgress renders the component 1`] = `
-<Fragment>
-  <Component
-    style={
+exports[`AccountCreationProgress final state - account ready 1`] = `""`;
+
+exports[`AccountCreationProgress initial state - before start account setup 1`] = `""`;
+
+exports[`AccountCreationProgress waiting for setup - waiting for ETH funding 1`] = `
+<Component
+  style={
+    Object {
+      "flexDirection": "row",
+    }
+  }
+>
+  <Component>
+    Funding account with ETH...
+  </Component>
+  <LinkToEtherscan
+    action={
       Object {
-        "flexDirection": "row",
+        "getTransaction": [Function],
       }
     }
-  >
-    <Component>
-      Funding account with ETH...
-    </Component>
-    <LinkToEtherscan
-      action={
-        Object {
-          "getTransaction": [Function],
-        }
+  />
+</Component>
+`;
+
+exports[`AccountCreationProgress waiting for setup - waiting for MANA funding 1`] = `
+<Component
+  style={
+    Object {
+      "flexDirection": "row",
+    }
+  }
+>
+  <Component>
+    Funding account with MANA...
+  </Component>
+  <LinkToEtherscan
+    action={
+      Object {
+        "getTransaction": [Function],
       }
-    />
+    }
+  />
+</Component>
+`;
+
+exports[`AccountCreationProgress waiting for setup - waiting for MANA funding and marketplace approval 1`] = `
+<Component
+  style={
+    Object {
+      "flexDirection": "row",
+    }
+  }
+>
+  <Component>
+    Funding account with MANA and approving marketplace...
   </Component>
-</Fragment>
+  <LinkToEtherscan
+    action={
```

Comment:
Not sure which action this will be. Maybe we can test which action it's linking to in this case (`Funding account with MANA and approving marketplace`) somehow, perhaps via a `LinkToEtherscan` test?

Reply:
I think this is the last thing we need to figure out before merging this.

Reply:
Sorry, but I didn't get it.

Reply:
It's the same thread/topic as this one https://github.com/tasitlabs/tasit/pull/260#issuecomment-480955462

Reply:
(And if we were to decide to hide it or show both actions, we'd want the test to reflect that)

Reply:
The `LinkToEtherscan` has an `action` as a prop and will render (action != `null`) it or not (action == `null`). The `AccountCreationProgress` component receives an `action` and a `status`. The snapshot is now show if the rendered action is or isn't `null`.

Reply:
What does the `AccountCreationProgress` component receive when there are two actions going on (`Funding account with MANA and approving marketplace`)?

I think our best bet is rendering two `AccountCreationProgress` components in that case and vertically stacking them on the screen. One for the mana funding, one for marketplace approval.

That way we can use `LinkToEtherscan` and `AccountCreationProgress` components as-is without modifying them.

Reply:
Hmmm, for now, the state is modeling one status/step per time:
```javascript
  state = {
    account: null,
    creationStatus: NOT_STARTED,
    creationActions: {},
  },
```

The `AccountCreationProgress` is showing message/link based on the current `creationStatus`. So what is happening now with paralelism is that one actions is overriding the other.
We can change the `creationStatus` prop from reducer to be an array, that will be a way to have the paralelism on the UI.

Reply:
Done

Reply:
Thanks! Resolving